server/src/test/java/org/elasticsearch/common/time/DateFormattersTests.java

```diff
@@ -378,7 +377,7 @@ public void testFractionalSeconds() {
         }
     }
 
-    public void testCamelCaseDeprecation() {
+    public void testCamelCaseRemoval() {
```

Comment:
When would we ever remove this test? I'm not sure we need negative assertions like this.

Reply:
agree, there is no point to have this test anymore.